### PR TITLE
AEA-1500 integration test and docs for simple aggregation skill

### PR DESCRIPTION
**Pull request by jrriehl**
## Proposed changes

Add an end-to-end integration test where 4 AEAs retrieve a coin price and take the average.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes and CI passes too
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/test_packages/test_skills_integration/test_simple_aggregation.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                "Start processing messages...",
+                "Fetching data from",
+                "Observation: {'price': {'value': ",
+                "found agents=",
+                "sending observation to peer=",
+                "received observation from sender=",
+                "Observations:",
+                "Average:",
+            )
+            missing_strings = self.missing_from_output(
```
Comment (DavidMinarsch) [inline tests/test_packages/test_skills_integration/test_simple_aggregation.py:187]: I'd split this up; first make sure all agents launch (so line 167-170), then check for the rest.

File: tests/test_packages/test_skills_integration/test_simple_aggregation.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+            )
+            setting_path = (
+                "vendor.fetchai.connections.http_server.config.target_skill_id"
+            )
+            self.set_config(setting_path, "fetchai/advanced_data_request:0.4.0")
+            self.set_config(
+                "vendor.fetchai.skills.simple_aggregation.models.strategy.args.quantity_name",
+                "price",
+            )
+
```
Comment (DavidMinarsch) [inline tests/test_packages/test_skills_integration/test_simple_aggregation.py:110]: Can you do a diff check to the fetched version too please?

**Comment by jrriehl** [comment]
I just realized that `aggregation` service is a very general specifier so that if multiple users run this demo, they will all aggregate with each other, which is kind of cool except that it doesn't matter what quantity they are aggregating. Probably we should add a standard way to specify the a unique identifier for the aggregators. This would fit it pretty nicely with the next step of adding "oracle requests".

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**